tests/unit_tests/c_wrappers/res/enkf/test_res_config.py

```diff
@@ -427,18 +427,18 @@ def test_res_config_dict_constructor(setup_case):
         ConfigKeys.MAX_RESAMPLE: 1,  # model
         ConfigKeys.TIME_MAP: "../input/refcase/time_map.txt",  # model
         ConfigKeys.INSTALL_JOB: [
-            {
-                ConfigKeys.NAME: "SNAKE_OIL_SIMULATOR",
-                ConfigKeys.PATH: "../../snake_oil/jobs/SNAKE_OIL_SIMULATOR",
-            },
-            {
-                ConfigKeys.NAME: "SNAKE_OIL_NPV",
-                ConfigKeys.PATH: "../../snake_oil/jobs/SNAKE_OIL_NPV",
-            },
-            {
-                ConfigKeys.NAME: "SNAKE_OIL_DIFF",
-                ConfigKeys.PATH: "../../snake_oil/jobs/SNAKE_OIL_DIFF",
-            },  # site
+            (
+                "SNAKE_OIL_SIMULATOR",
+                "../../snake_oil/jobs/SNAKE_OIL_SIMULATOR",
+            ),
+            (
+                "SNAKE_OIL_NPV",
+                "../../snake_oil/jobs/SNAKE_OIL_NPV",
+            ),
+            (
+                "SNAKE_OIL_DIFF",
+                "../../snake_oil/jobs/SNAKE_OIL_DIFF",
+            ),  # site
```

Comment:
This is :exploding_head:! Very nice! It made me realize that I was thinking about handling the difference between the config_dict resulting from converting a config_content object to a dictionary and a user-defined config_dict in a naive manner

Also, why is `# site ` there/needed?

Reply:
I have no idea what that comment means. I will remove it

Reply:
Figured it out. These comments seem to indicate what config they belong to, ie. in this case it indicated that it belonged to `site_config.py` . That is however outdated, so I am removing it. Moral of the story: don't have comments talk about stuff that is defined elsewhere.